experiment/.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
Added eslint to autoformat in VS Code. No need to check formatting in 3rd party dependencies or final minified code.